src/main/java/com/github/manolo8/darkbot/gui/titlebar/ExtraButton.java

```diff
@@ -62,8 +63,15 @@ public static void setExtraDecorations(Stream<ExtraMenuProvider> provider, Featu
             list.add(createSeparator("plugins"));
 
             PLUGINS.forEach((plugin, features) -> {
-                list.add(createMenu(plugin.getName(), 
-                    features.stream().flatMap(f -> f.getExtraMenuItems(main).stream())));
+                features.stream()
+                        .filter(f -> !f.shouldBeInsideMenu())
+                        .flatMap(f -> f.getExtraMenuItems(main).stream())
+                        .collect(Collectors.toCollection(() -> list));
+
+                list.add(createMenu(plugin.getName(),
+                    features.stream()
+                            .filter(ExtraMenuProvider::shouldBeInsideMenu)
+                            .flatMap(f -> f.getExtraMenuItems(main).stream())));
             });
```

Comment:
You should filter auto-submenued previous to this, directly on the setExtraDecorations method, only submit to the PluginExtraMenuProvider.PLUGINS the providers that ask to submenu.
`if (pl != null && extra.autoSubmenu()) {`